From: Etienne Millon <etienne.millon@gmail.com>
Date: Sun, 18 Dec 2011 22:45:03 +0100
Subject: Fix typo threshold

Glyr source code contains several occurrences of a typo on "threshold".
---
 lib/types.h       |    2 +-
 src/glyrc/glyrc.c |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/lib/types.h b/lib/types.h
index c3286c9..86ed1b1 100644
--- a/lib/types.h
+++ b/lib/types.h
@@ -285,7 +285,7 @@ typedef struct _GlyrDatabase {
 * @number: The maximum number of items to get; 0 -> inf
 * @plugmax: Max number of items per provider.
 * @verbosity: How verbose this query should be treated.
-* @fuzzyness: Max. treshold for levenshtein.
+* @fuzzyness: Max. threshold for levenshtein.
 * @img_min_size: Min. size in pixels an image may have.
 * @img_max_size: Min. size in pixels an image may have.
 * @parallel: Max. number of parallel queried providers.
diff --git a/src/glyrc/glyrc.c b/src/glyrc/glyrc.c
index 7900ac0..0053d1e 100644
--- a/src/glyrc/glyrc.c
+++ b/src/glyrc/glyrc.c
@@ -165,7 +165,7 @@ void help_short(GlyrQuery * s)
             IN"-n --number           Integer: Download max. <n> items. Amount of actual downloaded items may be less.\n"
             IN"-t --lang             String: Language settings. Used by a few getters to deliever localized data. Given in ISO 639-1 codes like 'de'\n"
             IN"-o --only-lang        Only use providers that offer language specific data, this only applies for text items.\n"
-            IN"-f --fuzzyness        Integer: Set treshold for level of Levenshtein algorithm.\n"
+            IN"-f --fuzzyness        Integer: Set threshold for level of Levenshtein algorithm.\n"
             IN"-q --qsratio          Float: How to weight quality/speed; 1.0 = full quality, 0.0 = full speed.\n"
             IN"-x --plugmax          Integer. Maximum number od download a plugin may deliever. Use to make results more vary.\n"
             IN"-v --verbosity        Integer. Set verbosity from 0 to 4. See --usage for details.\n"
-- 
